config/initializers/asset_sync.rb

```diff
@@ -0,0 +1,28 @@
+AssetSync.configure do |config|
+  config.fog_provider = 'AWS'
+  config.aws_access_key_id = ENV['AWS_ACCESS_KEY_ID']
+  config.aws_secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.